tests/lib/rules/id-length.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @fileoverview Tests for id-length rule.
+ * @author Burak Yigit Kaya
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var eslint = require("../../../lib/eslint"),
+    ESLintTester = require("eslint-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+var eslintTester = new ESLintTester(eslint);
+eslintTester.addRuleTest("lib/rules/id-length", {
+    valid: [
+        "var xyz;",
+        "var xy = 1;",
+        "function xyz() {};",
+        "function xyz(abc, de) {};",
+        "var obj = { abc: 1, de: 2 };",
+        "var obj = { 'a': 1, bc: 2 };",
+        "var obj = {}; obj['a'] = 2;",
+        "abc = d;",
+        "try { blah(); } catch (err) { /* pass */ }",
+        "var handler = function ($e) {};",
+        "var _a = 2",
+        "var _ad$$ = new $;",
+        "var xyz = new ΣΣ();",
+        { code: "var x = Foo(42)", args: [1, {"min": 1}] },
```

Comment:
Can you use `options` instead of args? We are deprecating args. `options` is the same except you don't need the first number (that's not actually used anywhere)